google-cloud-spanner-hibernate-testing/src/main/java/org/hibernate/userguide/model/Person.java

```diff
@@ -77,12 +77,12 @@
         name = "find_person_by_name",
         query =
             "SELECT " +
-            "   p.id AS \"id\", " +
```

Comment:
Another thing only we don't support?

Reply:
Yes I see the usual databases support this for aliases that have spaces. 

Here is the issue for keeping a list of these sorts of things until we have a ref doc https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/issues/47

Reply:
@meltsufin @dzou any outstanding change requests for this test file?

Reply:
Nope, I am good.